DuckDuckGo/PrivacyIconView.swift

```diff
@@ -54,7 +54,7 @@ class PrivacyIconView: UIView {
         updateAccessibilityLabels(for: icon)
     }
     
-    func loadAnimations(for theme: Theme, animationCache cache: AnimationCacheProvider = LRUAnimationCache.sharedCache) {
+    func loadAnimations(for theme: Theme, animationCache cache: AnimationCacheProvider = DefaultAnimationCache.sharedCache) {
```

Comment:
Lottie warning: 'LRUAnimationCache' is deprecated: Use DefaultAnimationCache instead, which is thread-safe and automatically responds to memory pressure.